.github/workflows/build-nflow-explorer.yaml

```diff
@@ -0,0 +1,35 @@
+name: Build and test nflow-explorer
```

Comment:
I think we should drop nflow-explorer, so this would not be needed

Reply:
dropped